src/repositories/happening-repo.js

```diff
@@ -14,6 +14,18 @@ class HappeningRepo {
 
     return result.rowCount > 0
   }
+  
+  static async remove ({id, accountId}) {
+    const result = await slonik.query(sql`
+      DELETE FROM happenings
+      WHERE
+      id=(${id})
+      AND
+      account_id=(${accountId});
```

Comment:
we shouldn't use accountId here, the interactor is responsible to check if such action is permitted